src/main/resources/plugin.yml

```diff
@@ -23,6 +23,11 @@ permissions:
     children:
       perworldinventory.bypass.gamemode: true
       perworldinventory.bypass.world: true
+  perworldinventory.useLastPosition.*:
+    default: true
+    children:
+      perworldinventory.useLastPosition.worldInGroup: true
+      perworldinventory.useLastPosition.locationInWorld: true
```

Comment:
This is not reflecten in code.

Switch to override rather than enable for consistency with other code.

Add permissions to override within group world change location redirect